pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -442,179 +492,191 @@ export class Benchmark {
         await saveSnap(this._testDirname, `[${this._poolType} - ${poolTag}] add liquidity proportional`, [receipt]);
       });
 
-      it(`measures gas (unbalanced) (${poolTag})`, async () => {
-        const exactAmountsIn = Array(poolInfo.poolTokens.length)
-          .fill(TOKEN_AMOUNT)
-          .map((amount, index) => BigInt(amount / BigInt(index + 1)));
+      (this._settings.disableUnbalancedLiquidityTests ? it.skip : it)(
```

Comment:
ah, `skip` didn't work inside `it`, did it?

This is fine but it's a bit complicated to read. The answer linked above mentions that if you declare the function explicitly it should work:

```
it(`measures gas (unbalanced) (${poolTag})`, async function {
    if (this._settings.disableUnbalancedLiquidityTests) {
        this.skip();
    }
    ...
});
```